### ISSUE TITLE
[CH] support split_part function

### ISSUE THREAD

**Issue by taiyang-li** [apache/incubator-gluten#6807]
### Description

s

**Comment by soupam05** [comment]
Can I pickup this?

**Comment by navaneethsujith09** [comment]
@taiyang-li Is this issue done already? If not, do you think you can assign this issue to me?

**Comment by taiyang-li** [comment]
@navaneethsujith09  no, current issue is still pending. Of course, I can assign you to you. Thank you for your contribution
